chargeOclock/Source/DataSources/ClientsDataSource.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ClientsDataSource.swift
+//  chargeOclock
+//
+//  Created by Goran Blažič on 08/01/2017.
+//  Copyright © 2017 goranche.net. All rights reserved.
+//
+
+import UIKit
+
+class ClientsDataSource: NSObject, UITableViewDataSource {
+
+	let clients: [Client]
+	let reuseIdentifier: String
```

Comment:
it doesn't need to, but since it's a constant that's provided by the code instantiating this class, it doesn't hurt either and might come in handy (especially since the reuseIdentifier has a default value in the constructor, so the calling code might not even be passing it in)